GitUI/UserControls/FileStatusList.cs

```diff
@@ -1409,8 +1409,10 @@ private void FileStatusListView_ContextMenu_Opening(object sender, CancelEventAr
                 cm.Items.Add(_NO_TRANSLATE_openInVisualStudioMenuItem);
             }
 
-            _NO_TRANSLATE_openInVisualStudioMenuItem.Visible = _openInVisualStudioSeparator.Visible = VisualStudioIntegration.IsVisualStudioRunning;
-            _NO_TRANSLATE_openInVisualStudioMenuItem.Enabled = File.Exists(SelectedItemAbsolutePath);
+            bool fileExists = File.Exists(SelectedItemAbsolutePath);
+            _NO_TRANSLATE_openInVisualStudioMenuItem.Enabled = fileExists;
```

Comment:
A specific reason why you kept the 'Enabled' whereas the item won't be visible?

Reply:
Please see the following lines... (I have changed the sequence.)

Reply:
Remove IsVisualStudioRunning too?

Reply:
> Remove IsVisualStudioRunning too?

turned into `IsVisualStudioInstalled`

> A specific reason why you kept the 'Enabled' whereas the item won't be visible?

I am used to set both because I am not sure whether `Visible = false` overrides `Enabled`.
In theory, mnemonics could still be active if the item is enabled but invisible.